core/dbt/include/global_project/macros/materializations/models/materialized_view/create_materialized_view_as.sql

```diff
@@ -0,0 +1,11 @@
+-- MATERIALIZED_VIEW IMPLMENTATION
+{% macro create_materialized_view_as(relation, sql, config) %}
```

Comment:
What does `config` get used for here? Will that be for the `on_configuration_change` piece?

Reply:
sorry that is leftover from experimental implementation I believe similar to `config.get` which seems to be more of the norm for us so will remove